lib/src/native/rtc_video_view_impl.dart

```diff
@@ -53,13 +55,13 @@ class RTCVideoView extends StatelessWidget {
               child: Transform(
                 transform: Matrix4.identity()..rotateY(mirror ? -pi : 0.0),
                 alignment: FractionalOffset.center,
-                child: videoRenderer.textureId != null &&
-                        videoRenderer.srcObject != null
-                    ? Texture(
-                        textureId: videoRenderer.textureId!,
-                        filterQuality: filterQuality,
-                      )
-                    : Container(),
+                child:
+                    videoRenderer.textureId != null && videoRenderer.renderVideo
```

Comment:
@ycherniavskyi Good idea to use the renderVideo bool to perform the check.  After checking the implementation of the RTCVideoRenderer of the Native and Web, I think we can safely move the  video.textureId != null under the umbrella of renderVideo to make the code more readable for the widget.  

`bool get renderVideo => srcObject != null && _textureId != null;`

Is this sounds resonable?

Reply:
I agree. Also, I wouldn't say I like this extra condition in the native widget.